playground/infrastructure/helm-playground/templates/deployment-frontend.yml

```diff
@@ -34,6 +34,25 @@ spec:
        imagePullPolicy: {{ .Values.image.pullPolicy }}
        ports:
         - containerPort: 8080
+       livenessProbe:
+         httpGet:
+           path: /liveness
+           port: 8080
+         initialDelaySeconds: 3
+         periodSeconds: 3
+       readinessProbe:
+         httpGet:
+           path: /readiness
+           port: 8080
+         initialDelaySeconds: 3
+         periodSeconds: 3
+       resources:
+         requests:
+           memory: "50Mi"
+           cpu: "250m"
+         limits:
+           cpu: "500"
+           memory: "500Mi"
```

Comment:
How was the values `50Mi`, `250m`, `500`, and `500Mi` determined?  I wouldn't know myself and not a blocker for this PR.  I was just curious and thank you.

Reply:
We did some tests thru the Apache JMeter. These resources are enough for simple tasks, and they will be autoscaled once loading will be increased